WireDomain/Sources/WireDomain/Notifications/Components/NotificationServiceExtensionFlow.swift

```diff
@@ -35,6 +37,11 @@ final class NotificationServiceExtensionFlow: BootstrapComponent {
         self.contentHandler = contentHandler
 
         let infoDictionary = Bundle.main.infoDictionary
+
+        guard let currentAppVersion = infoDictionary?["CFBundleShortVersionString"] as? String else {
+            throw Failure.missingAppGroupID
```

Comment:
issue: don't you mean?
```suggestion
            throw Failure.missingCurrentAppVersion
```

Reply:
question: so it means no notifications until user opened the app, right?

Reply:
thought: I wonder if we should show a specific notification to the user. Please open the app (it applies also to DB migration)

Reply:
yes fixed: 5ca2880a12e75daf30da66cfe08fd99cd85711fd

Reply:
> question: so it means no notifications until user opened the app, right?

How so? The version comes from the bundle